src/disjoint_mut.rs

```diff
@@ -273,6 +274,7 @@ impl<T: ?Sized + AsMutPtr> DisjointMut<T> {
     ///
     /// [`index`]: DisjointMut::index
     /// [`index_mut`]: DisjointMut::index_mut
+    #[inline(always)] // Need to inline to see bounds checks in order to potentially elide them.
```

Comment:
I mentioned this on another review, but we should use `#[inline]` instead.

Reply:
Fixed in [`e25ab1c..3b2e057`](https://github.com/memorysafety/rav1d/compare/e25ab1c181c7bf6103ba2815937939ae70b37a94..3b2e057496de15d60bc039e9af9c288224bfa1ae).